src/network/SessionService.cpp

```
@@ -25,26 +47,119 @@ SessionService::SessionService(AuthenticationService *authService, QObject *pare
+{
+    return authenticationSessionMode()
+        ? QStringLiteral("Authentication Sessions")
+        : QStringLiteral("Playback Sessions");
+}
+
+QString SessionService::sessionTypeDescription() const
+{
+    return authenticationSessionMode()
+        ? QStringLiteral("Devices signed in to this account")
+        : QStringLiteral("Devices currently connected for playback");
+}
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If Q_UNUSED is a macro then please configure it.

(unknownMacro)